### ISSUE TITLE
mysql8 sql syntax REPLACE INTO throwing errors after v24.0.3

### ISSUE THREAD

**Issue by davidlhw** [dbeaver/dbeaver#26842]
## Context

The following bug does not occur in CE v24.0.2

### Table Schema
```sql
-- Table definition
CREATE TABLE `Table` (
  `col1` varchar(128) NOT NULL,
  `col2` varchar(45) NOT NULL,
  `col3` int NOT NULL DEFAULT '0',
  `col4` varchar(10) NOT NULL DEFAULT '*',
  `col5` text NOT NULL,
  `updated_at` timestamp NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
  PRIMARY KEY (`col1`,`col3`,`col2`,`col4`),
  KEY `some_key_name_idx` (`col3`,`col4`),
  KEY `some_key_name_2_idx` (`col2`,`col3`,`col4`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb3;
```

### In custom SQL Script, created by clicking on Scripts tab (`^]`)
```sql
INSERT INTO `dbName`.Table (col1,col2,col3,col4,col5) VALUES 
  ('some_value_1','some_value_2',0,'*','some_value_5');

REPLACE INTO `dbName`.Table (col1,col2,col3,col4,col5) VALUES
  ('some_value_1','some_value_2',0,'*','some_new_value_5');
```

## when `REPLACE INTO` query is at the top of the script
![screenshot_20240426_161649](https://github.com/dbeaver/dbeaver/assets/13061926/e665be57-9d52-401d-b428-98b5f6b213ed)

## when `REPLACE INTO` query is below another query in the same script

The `REPLACE INTO` query gets evaluated together with the above query when executing a query under the cursor or clicking `^⏎` 

![screenshot_20240426_161443](https://github.com/dbeaver/dbeaver/assets/13061926/37476b36-d4bb-46c6-b94c-68649d49f826)

```txt
SQL Error [1064] [42000]: You have an error in your SQL syntax; check the manual that 
corresponds to your MySQL server version for the right syntax to use near 'REPLACE INTO 
`dbName`.Table at line 5

Error position: line: 4
```

### DBeaver Version

Community Edition Version 24.0.3.202404211624

### Operating System

macOS Sonoma 14.0 (23A344)

### Database and driver

![image](https://github.com/dbeaver/dbeaver/assets/13061926/2c7e3665-3600-4945-87c4-693b73ef3417)

### Steps to reproduce

1. use v24.0.3
2. create table per above schema
3. insert a row per above query
4. attempt to replace values in the row per above query
5. see error

**Comment by davidlhw** [comment]
potentially related to https://github.com/dbeaver/dbeaver/issues/26447 & https://github.com/dbeaver/dbeaver/issues/25061

**Comment by E1izabeth** [comment]
Thank you for bug report. It's a duplicate of #26416
We're working on fix

As a workaround, you could try switching `Blank line is statement delimiter` setting to `Always`
![image](https://github.com/dbeaver/dbeaver/assets/28875055/be8457a0-d62c-4acb-ac2f-67ce518cf3f5)

**Comment by davidlhw** [comment]
hey @E1izabeth, thanks for the work around. it did solve some of the problem but there are still lingering errors when trying to run multi-lined queries here and there in a large sql script

downgrading to v24.0.2 seems to be a simpler solution

**Comment by E1izabeth** [comment]
Does downgrading help?
Could you clarify the cases where it doesn't work as expected?

**Comment by davidlhw** [comment]
downgrading to v24.0.2 helped, even with the following issues -

recalling from memory, a few cases that didn't work well even with blank line as delimiter set to `always`:
- `REPLACE INTO` query under a large multi-line query
- multi-line `SELECT` query under another query works, but trying to filter the `SELECT` query result failed

i'm uncertain if this is the direct cause of the same issue as i just tried upgrading back to v24.0.3 again to recreate the issue but it no longer breaks

thanks for the fix 🔥

**Comment by E1izabeth** [comment]
Actually, we haven't delivered a fix yet, so I suppose you reinstalled the same version with the bug.
The fix will be included in 24.0.4 version and also available in Early Access version (https://dbeaver.io/files/ea/) tomorrow
